SPECS/libsoup/libsoup.spec

```diff
@@ -1,41 +1,42 @@
 %define BaseVersion 3.0
```

Comment:
shouldn't you remove that (or update it)

Reply:
actually, some of the files still have this baseversion (3.0) in the filename. So I kept it